components/analytics/team_analytics/index.ts

```diff
@@ -1,20 +1,20 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {connect} from 'react-redux';
-import {bindActionCreators} from 'redux';
-import {getTeams} from 'mattermost-redux/actions/teams';
-import {getProfilesInTeam} from 'mattermost-redux/actions/users';
-import {getTeamsList} from 'mattermost-redux/selectors/entities/teams';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { getTeams } from 'mattermost-redux/actions/teams';
+import { getProfilesInTeam } from 'mattermost-redux/actions/users';
+import { getTeamsList } from 'mattermost-redux/selectors/entities/teams';
 
 import BrowserStore from 'stores/browser_store.jsx';
-import {getCurrentLocale} from 'selectors/i18n';
+import { getCurrentLocale } from 'selectors/i18n';
 
-import TeamAnalytics from './team_analytics.jsx';
+import TeamAnalytics from './team_analytics';
 
 const LAST_ANALYTICS_TEAM = 'last_analytics_team';
 
-function mapStateToProps(state) {
+function mapStateToProps(state: any) {
```

Comment:
We have a type definition for this called `GlobalState`, found in the `types/store` module